app/webpacker/components/CompetitionsOverview/CompetitionsFilters.js

```diff
@@ -285,7 +280,7 @@ function PastCompYearSelector({ filterState, dispatchFilter }) {
           >
             {I18n.t('competitions.index.all_years')}
           </Dropdown.Item>
-          {PAST_YEARS_WITH_COMPETITIONS.map((year) => (
+          {YEARS_WITH_PAST_COMPETITIONS.map((year) => (
```

Comment:
Maybe `YEARS_LIST_OF_PAST_COMPETITIONS` or `YEARS_OF_PAST_COMPETITIONS`?

Reply:
Hm, that's why they say "naming stuff is hard" in software development :smile: 
I'm fine with either and I won't block this PR just because of some minor naming issue of a constant used to populate a dropdown. If it was a more central value, amybe I'd argue more, but honestly for a helper variable I am 50/50 on spending time to perfect it on one hand and just getting the task done on the other hand.